docs/suspensive.org/src/pages/docs/_meta.en.json

```diff
@@ -17,6 +17,12 @@
       "collapsed": true
     }
   },
+  "jotai": {
+    "title": "@suspensive/jotai",
+    "theme": {
+      "collapsed": true
+    }
+  },
```

Comment:
Could you next of react-query and don't collapsed please.

```suggestion
  "jotai": "@suspensive/jotai (WIP)",
```

Reply:
Could you right next of react-query plesae. jotai's priority could be next of react-query

Reply:
@manudeli OMG! I'm sorry. I missed that.

Reply:
<img width="236" alt="image" src="https://github.com/user-attachments/assets/70edb80d-fe28-4575-ad72-95f076514646">

I've fixed it! Sorry for the inconvenience.

Reply:
I think @suspensive/jotai is already clearly indicating that this is an experimental feature on the page, so I removed the "(WIP)" text because it seemed unnecessary in the title.